README.md

```diff
@@ -63,7 +63,7 @@ For the small price of 7kb gziped. &nbsp;&nbsp; [🏁Get started now](docs/getti
 - [x] 🧘 Flexible to fit any API design (one size fits all)
 - [x] 🌳 Tree-shakable (only use what you need)
 - [ ] 🚯 Pluggable garbage collection policy
-- [ ] 🔁 Subscriptions
+- [x] 🔁 Subscriptions
```

Comment:
nit: maybe move this above `Pluggable garbage collection policy`, so all of the features that have been completed are grouped together

Reply:
👍